cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,159 @@
-/// <reference types='cypress' />
+describe("Web Tables page", () => {
+  let user;
 
-describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
+    cy.visit("https://demoqa.com/webtables");
+  });
+
+  it("should have the pagination", () => {
```

Comment:
with this test you only check the presence of the pagination but don't check the functionality. Check pages changing

Reply:
done